doc/Makefile.am

```diff
@@ -13,4 +13,11 @@ DEFAULT_ERROR_DIR       = $(datadir)/errors
 
 EXTRA_DIST = \
 	debug-messages.dox \
-	debug-sections.txt
+	debug-sections.txt \
+	SPONSORS.in
+
+data_DATA = SPONSORS
+CLEANFILES = $(data_DATA)
+
+SPONSORS: SPONSORS.in
+	sed -e 's/@Squid-[0-9\.]*://' <$< >$@
```

Comment:
The tags are there for editors use maintaining the sort by sponsor contribution time. They are still very useful for that purpose.

If anything I am minded to leave them in and drop the whole generation of `SPONSORS` thin in favour of a static file.

Reply:
> The tags are there for editors use maintaining the sort by sponsor contribution time. They are still very useful for that purpose.

My change request was written with those two facts in mind.

> If anything I am minded to leave them in and drop the whole generation of `SPONSORS` thing in favour of a static file.

IIRC, we have discussed this before: Exposing version tags or similar information to SPONSORS readers is a good idea, but doing so _properly_ requires a redesign to avoid creating a false impression that some of the entries are version-specific (or duplicating those entries across many versions, which would be even worse!). The current awkward state was a compromise we reached back then, probably either to avoid dealing with that redesign or while waiting for it to happen.

FWIW, I do support redesigning SPONSORS layout to avoid its generation while adding support for the multi-version acknowledgments. I can suggest a specific format that would address both goals if others support that transition in principle.

Reply:
> From which we are left to conclude that you are either not caring about the extra work/time/difficulty your request forces on the editor/maintainer of that document (me primarily), or you maliciously intend that result.

Both conclusions would be wrong. BTW, this change request is a suggestion that does not force anything "extra" on anybody.

> Your idea of "properly" has not yet been defined.

This change request contains some relevant ideas, but flashing out details requires your support for those ideas (in principle). Without it, there is no point in discussing implementation details! All the extra work mentioned in my change request (and its followup) was explicitly preconditioned on support for those ideas from others. FWIW, the word "properly" in this context can be interpreted as "without serious flaws like those mentioned later in the same sentence".